Readme.md

```diff
@@ -2,260 +2,308 @@
 
 [![Gitter](https://badges.gitter.im/rust-sgx-sdk/community.svg)](https://gitter.im/rust-sgx-sdk/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-# Rust SGX SDK
-Rust SGX SDK helps developers write Intel SGX applications in Rust programming language. [[CCS'17 Paper pdf]](documents/ccsp17.pdf) [[CCS'19 Paper pdf]](https://dingelish.com/ccs19.pdf)
+# Apache Teaclave Rust-SGX SDK
+The Teaclave Rust-SGX SDK helps developers write Intel SGX applications in the Rust programming language. <sup>[[1]](documents/ccsp17.pdf) [[2]](https://dingelish.com/ccs19.pdf)</sup> Please read our documentation for SDK crates which can be found [here](https://dingelish.github.io/) or can be autogenerated with `cargo doc`.
```

Comment:
The API docs of Teaclave SGX SDK is hosted in Teaclave's homepage: https://teaclave.apache.org/api-docs/sgx-sdk/sgx_tstd/index.html. I suggest to put this link.